### PR TITLE
Priors and posteriors should be Parameter attributes

### PR DESCRIPTION
**Pull request by nden**
Not sure why they are not. Perhaps we forgot to change them?

### CODE REVIEW

File: astropy/modeling/parameters.py [hunk 1]
```diff
@@ -433,50 +432,30 @@ def size(self):
...
     def fixed(self):
         """
         Boolean indicating if the parameter is kept fixed during fitting.
         """
-
         return self._fixed
 
     @fixed.setter
     def fixed(self, value):
-        """Fix a parameter"""
```
Comment (perrygreenfield) [inline astropy/modeling/parameters.py]: Is this error checking not needed for a reason? (e.g., handy to supply other False or True equivalents?)

Comment (nden) [reply astropy/modeling/parameters.py]: I'm not sure how this happened. Will put it back.

**Comment by nden** [comment]
@bsipocz Changed the milestone to 4.0. 
Hm, there's a change log in 4.0.1. Should i do anything about it?

**Comment by bsipocz** [comment]
No worries about the changelog, I'll fix it up at release time and will propagate the changes back to master.

**Comment by bsipocz** [comment]
However, just one thing. This changelog is still valid for 4.0, or would this be an affect-dev change that could have happened during the big refactor?

**Comment by nden** [comment]
It's a change that happened during the big refactor. So it's Affect-dev. Thanks!

**Pull request merged**